### ISSUE TITLE
Armor Progression in wrong format presentation

### ISSUE THREAD

**Issue by Yyl06** [Yyl06/hypixel-web-guide#3]
<img width="847" height="880" alt="Image" src="https://github.com/user-attachments/assets/dcc12ff5-7569-4394-9216-2007c063086f" />

The section of this part should be paragraph-based instead of being border-based
